<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PCSClass</class>
 <widget class="QMainWindow" name="PCSClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1135</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PCS</string>
  </property>
  <property name="windowIcon">
   <iconset resource="pcs.qrc">
    <normaloff>:/PCS/Resources/PCS/b.png</normaloff>:/PCS/Resources/PCS/b.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QWidget" name="widgetWin" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>450</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget{
	border: 2px solid white;
	border-radius: 10px;
	background: qlineargradient(x1:0, y1:0, x2:0, y2:1,stop:0 rgb(10,102,150), stop:1 rgb(0,0,0));
 }

</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QFrame" name="frameWin">
         <property name="maximumSize">
          <size>
           <width>500</width>
           <height>400</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(:/PCS/Resources/png/qidong.png);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1135</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionRecentfile"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu3D">
    <property name="title">
     <string>窗口</string>
    </property>
    <addaction name="actionNew3DView"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
    <addaction name="actionCloseAll"/>
    <addaction name="separator"/>
    <addaction name="actionTitle"/>
   </widget>
   <widget class="QMenu" name="menuDisplay">
    <property name="title">
     <string>显示</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuViewTool">
    <property name="title">
     <string>视图</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuDataTool">
    <property name="title">
     <string>点云数据处理</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuPointData">
    <property name="title">
     <string>激光点信息</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>点云数据读取</string>
    </property>
    <addaction name="actionUSBread"/>
    <addaction name="actionRSSread"/>
    <addaction name="actionWebread"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPointData"/>
   <addaction name="menu"/>
   <addaction name="menuDataTool"/>
   <addaction name="menuDisplay"/>
   <addaction name="menu3D"/>
   <addaction name="menuViewTool"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>19</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/PCS/Resources/png/statusbar.png);</string>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBarMainTool">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>29</height>
    </size>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
  </widget>
  <widget class="QToolBar" name="toolBarViewTool">
   <property name="minimumSize">
    <size>
     <width>29</width>
     <height>0</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QDockWidget" name="dockProperties">
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>230</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>180</width>
     <height>524287</height>
    </size>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTreeWidget" name="TableProperties">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <column>
        <property name="text">
         <string/>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockDBtree">
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>111</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>180</width>
     <height>524287</height>
    </size>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_11">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTreeView" name="TableDBtree_3"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockConsole">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>524287</width>
     <height>300</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetClosable|QDockWidget::DockWidgetFloatable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>输出</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_9">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QListWidget" name="Console">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>80</height>
        </size>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBarDataTool">
   <property name="windowTitle">
    <string>toolBar_3</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBarCameraTool">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset theme="open" resource="pcs.qrc">
     <normaloff>:/png/Resources/icon/open-file.ico</normaloff>:/png/Resources/icon/open-file.ico</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
   <property name="toolTip">
    <string>打开   </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset theme="save" resource="pcs.qrc">
     <normaloff>:/png/Resources/icon/save.ico</normaloff>:/png/Resources/icon/save.ico</iconset>
   </property>
   <property name="text">
    <string>保存 </string>
   </property>
   <property name="toolTip">
    <string>保存 </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionNew3DView">
   <property name="text">
    <string>3D</string>
   </property>
   <property name="toolTip">
    <string>3D</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F3</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>关闭当前视图</string>
   </property>
  </action>
  <action name="actionCloseAll">
   <property name="text">
    <string>关闭所有试图</string>
   </property>
   <property name="toolTip">
    <string>关闭所有试图</string>
   </property>
  </action>
  <action name="actionTitle">
   <property name="text">
    <string>视图标题</string>
   </property>
   <property name="toolTip">
    <string>视图标题</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>另存为</string>
   </property>
   <property name="toolTip">
    <string>另存为</string>
   </property>
  </action>
  <action name="actionRecentfile">
   <property name="text">
    <string>最近文件</string>
   </property>
   <property name="toolTip">
    <string>最近文件</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>帮助</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="actionUSBread">
   <property name="text">
    <string>USB读取</string>
   </property>
  </action>
  <action name="actionWebread">
   <property name="text">
    <string>网络接口读取</string>
   </property>
  </action>
  <action name="actionRSSread">
   <property name="text">
    <string>RSS端口读取</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="pcs.qrc"/>
 </resources>
 <connections/>
</ui>
